src/swdmt/redistricting/Redistrictor.java

```diff
@@ -107,13 +107,11 @@ public Region region() {
      * specified size from a given region.
      * If the region is smaller than the specified size,
      * then a single district is returned.
-     * Otherwise, creates a set of all districts of
-     * approximately equal size; that is, each district's
-     * size is within ±1 of the district size parameter.
+     * Otherwise, a set is created that contains all 
```

Comment:
This looks good, just one minor edit request: Change "a set is _created_" to "a set is _returned_"